build/teamcity-nightly-roachtest.sh

```diff
@@ -78,15 +78,9 @@ trap upload_stats EXIT
 # Set up the parameters for the roachtest invocation.
 PARALLELISM=16
 CPUQUOTA=1024
-ZONES=""
 TESTS=""
 case "${CLOUD}" in
   gce)
-    # We specify --zones below so that nodes are created in us-central1-b by
```

Comment:
why this change?

Reply:
This doesn't do anything. The logic to export `ZONES` was removed in https://github.com/cockroachdb/cockroach/commit/d60cf646978ceb1d4757322cdb43157681601636.